azext_iot/sdk/iothub/controlplane/__init__.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
```

Comment:
The iot hub control plane client needs to be removed from this PR.

Reply:
I will keep this in for development until at least the new sdk with cosmos db functionality comes out.